src/main/webapp/i18n/de/student-dashboard.json

```diff
@@ -10,14 +10,16 @@
             "cardNoExerciseLabel": "Keine Übung geplant",
             "cardExerciseLabel": "Nächste Übung:",
             "register": {
-                "signUp": "Für einen Kurs anmelden",
```

Comment:
The translations of these sections are a bit... interesting :D
Generally, it's called `registration`. In German, you call it `Anmeldung` while in English, you stay with `registration`.
However, some keys are called `signUp...` while you replaced others.

In the end, it's not even the correct term because it's not about registration (we use that term for account creation), but about enrollment. But that's not your fault.

What I do expect is that the naming is consistent. If you agree, I would be even happier if you named it correctly. But that's a big bonus and optional :D

Reply:
I changed the `register.signUp` translation string to `register.title`, but otherwise left my changes as before. Now it's consistently
- registration in the code and in English
- Anmeldung in German

I didn't want to change it to "enrol" because then I would have to change it in the code to be consistent between English & the code, which would just include too much files for now.